docs/data-sources/user.md

```diff
@@ -0,0 +1,35 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_user Data Source - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  Twingate users. For more information, see Twingate's documentation https://docs.twingate.com/docs/users.
+---
+
+# twingate_user (Data Source)
+
+Twingate users. For more information, see Twingate's [documentation](https://docs.twingate.com/docs/users).
+
+## Example Usage
+
+```terraform
+data "twingate_user" "foo" {
+  id = "<your user's id>"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `id` (String) The ID of the User
```

Comment:
Update to:

`...User. The ID for the User must be obtained from the Admin API.`

Reply:
updated